CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+# Code Ownership for buzzmobile.
+# Order matters. The last match is used, so more specific owners should be 
+# last.
+
+# Default owners/reviewers for anything.
+* @irapha @joshuamorton
```

Comment:
Since sahit will low key take over car stuff this semester, you should add him here too.